docs/spec-files/feature_discovery.md

```diff
@@ -52,15 +52,15 @@ A `discover-features/query` message looks like this:
     "body": {
         "queries": [
             { "feature-type": "protocol", "match": "https://didcomm.org/tictactoe/1.*" },
-            { "feature-type": "goal-code", "match": "aries.*" }
+            { "feature-type": "goal-code", "match": "org.didcomm.*" }
         ]
     }
 }
 ```
 
-Queries messages contain one or more **query objects** in the `queries` array. Each query essentially says, "Please tell me what features of type X you support, where the feature identifiers match this (potentially wildcarded) string." This particular example asks an agent if it supports any 1.x versions of the [tictactoe protocol](https://github.com/hyperledger/aries-rfcs/blob/main/concepts/0003-protocols/tictactoe/README.md), and if it supports any [goal codes](https://github.com/hyperledger/aries-rfcs/blob/main/concepts/0519-goal-codes/README.md) that begin with "aries.".
+Queries messages contain one or more **query objects** in the `queries` array. Each query essentially says, "Please tell me what features of type X you support, where the feature identifiers match this (potentially wildcarded) string." This particular example asks an agent if it supports any 1.x versions of the [tictactoe protocol](https://github.com/hyperledger/aries-rfcs/blob/main/concepts/0003-protocols/tictactoe/README.md), and if it supports any [goal codes](https://github.com/hyperledger/aries-rfcs/blob/main/concepts/0519-goal-codes/README.md) that begin with "org.didcomm.".
 
-Implementations of this protocol must recognize the following values for `feature-type`: `protocol`, `goal-code`, `gov-fw`, `didcomm-version`, and `decorator`/`header`. (The concept known as `decorator` in DIDComm v1 approximately maps to the concept known as `header` in DIDComm v2. The two values should be considered synonyms and must both be recognized.) Additional values of `feature-type` may be standardized by raising a PR against this RFC that defines the new type and increments the minor protocol version number; non-standardized values are also valid, but there is no guarantee that their semantics will be recognized.
+Implementations of this protocol must recognize the following values for `feature-type`: `protocol`, `goal-code`, `gov-fw`, `didcomm-version`, and `header`.  Additional values of `feature-type` may be used, and unrecognized values MUST be ignored.
```

Comment:
I think we should state the purpose of each of these values if we're going to state they must be recognized.

Reply:
discussed - language needs to be softened around goal code, maybe rmove gov-fw.